assets/app/view/game/route_selector.rb

```diff
@@ -149,16 +158,29 @@ def render
             ]),
             h(:tbody, trains),
           ]),
-          actions,
+          actions(render_halts),
         ])
       end
 
+      def halt_actions(route, revenue, subsidy)
+        change_halts = lambda do
+          route.cycle_halts
+          store(:selected_route, route)
+        end
+
+        [
+          revenue,
+          h(:br),
```

Comment:
i don't think we need a br, just use a style

Reply:
Ok, I'll have to figure out how I can use a style to place the button under the revenue.

Reply:
Changed to use div, which avoids the need for br